test/browser/wrapper.jsx

```diff
@@ -0,0 +1,181 @@
+/* global before, beforeEach, after, describe, expect, it */
+import Wrapper from '../../src/wrapper'
+
+const injectSelectToEnhanceIntoDOM = (element, id, name, options, selectedOption) => {
+  var $select = document.createElement('select')
+  if (id) {
+    $select.id = id
+  }
+  if (name) {
+    $select.name = name
+  }
+
+  for (var optionKey in options) {
```

Comment:
Not sure what you mean by duplication? I meant:

```js
Object.keys(options)
  .map(optionKey => {
    const option = document.createElement('option')
    option.value = optionKey
    option.text = options[optionKey]
    option.selected = (selectedOption === optionKey)
    return option
  })
  .forEach(option => $select.appendChild(option))
```

Now that I've written it I have a strong preference for this, because I realised it would keep our codebase for-loop-free™.